transformers/events/cat_file/box/transformer_test.go

```diff
@@ -0,0 +1,61 @@
+package box_test
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+
+	//"github.com/ethereum/go-ethereum/core"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/cat_file/box"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+
+	//"github.com/makerdao/vdb-mcd-transformers/transformers/events/cat_file/box"
+	//"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Cat file box transformer", func() {
+	var (
+		transformer = box.Transformer{}
+		db          = test_config.NewTestDB(test_config.NewTestNode())
+	)
+
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+	})
+
+	It("converts a box log to a model", func() {
+		models, toModelsErr := transformer.ToModels(constants.Cat110ABI(), []core.EventLog{test_data.CatFileBoxEventLog}, db)
+		Expect(toModelsErr).NotTo(HaveOccurred())
+
+		contractAddressID, msgSenderErr := shared.GetOrCreateAddress(test_data.Cat110Address(), db)
```

Comment:
:100: I like the way this reads much better. I was putting something like this in my tests - `shared.GetOrCreateAddress(test_data.CatFileBoxEventLog.Log.Address.Hex(), db)`